lib/lhm/locked_switcher.rb

```diff
@@ -23,12 +23,11 @@ class LockedSwitcher
 
     attr_reader :connection
 
-    def initialize(migration, connection = nil, retain_triggers = false)
```

Comment:
@bhuvanesh-sankar can we not instead make this as `retain_triggers = true`?
That is by default if the flag is not passed, it will be true and you would also have the option to pass it as false - retaining the original percona behavior. What do you think?

I don't see a use case though to not retain the triggers.

@ImAjayS Your thoughts?